homeassistant/components/swiss_public_transport/strings.json

```diff
@@ -49,12 +49,34 @@
       }
     }
   },
+  "services": {
+    "fetch_connections": {
+      "name": "Fetch Connections",
+      "description": "Fetch a list of connections from the swiss public transport api.",
+      "fields": {
+        "entity_id": {
+          "name": "List",
+          "description": "Swiss public transport sensor to fetch connections for."
+        },
+        "limit": {
+          "name": "Limit",
+          "description": "Number of connections to fetch from [1-15]"
+        }
+      }
+    }
+  },
   "exceptions": {
     "invalid_data": {
       "message": "Setup failed for entry {config_title} with invalid data, check at the [stationboard]({stationboard_url}) if your station names are valid.\n{error}"
     },
     "request_timeout": {
       "message": "Timeout while connecting for entry {config_title}.\n{error}"
+    },
+    "cannot_connect": {
+      "message": "Cannot connect to server.\n{error}"
+    },
+    "bad_config": {
+      "message": "Request failed due to bad config: Check at the [stationboard]({stationboard_url}) if your station names are valid.\n{error}"
```

Comment:
In that case the sensor itself would also be unavailable I guess so maybe bail out early if the sensor is not available as then this service call won't work either?

Reply:
True, with a bad config the sensor should never be available.

Reply:
@gjohansson-ST Is there a standard error to throw when a sensor is not available? I just came across `ServiceValidationError` but don't know if there is a standard `translation_key` for example? I tried to look for something like that in the code, but could not find it.

Reply:
https://developers.home-assistant.io/docs/internationalization/core/#exceptions